examples/zenohcxx/zenohc/z_get_channel_non_blocking.cxx

```diff
@@ -23,9 +23,8 @@
 #include <unistd.h>
 #endif
 
-#include "zenohcpp.h"
-
-using namespace zenoh;
+#include "zenohc.hxx"
+using namespace zenohc;
```

Comment:
Shouldn't it be `zenoh` namespace like in the other examples?

Reply:
No, this is specific zenohc example and this demonstrates how to directly use zenohcxx for zenohc. This is also mentioned in readme